--- CPP/7zip/7zip.mak	Mon Jun 23 11:56:48 2014
+++ CPP/7zip/7zip.mak	Mon Apr 27 15:38:13 2020
@@ -1,5 +1,4 @@
-OBJS = \
-  $O\StdAfx.obj \
+OBJS_NEEDING_PCH = \
   $(CURRENT_OBJS) \
   $(COMMON_OBJS) \
   $(WIN_OBJS) \
@@ -26,9 +25,11 @@
   $(ZIP_OBJS) \
   $(COMPRESS_OBJS) \
   $(CRYPTO_OBJS) \
-  $(C_OBJS) \
+  $(C_OBJS)
+
+OBJS = $(OBJS_NEEDING_PCH) \
   $(ASM_OBJS) \
-  $O\resource.res \
+  $O\resource.res

 !include "../../../Build.mak"

