--- CPP/Build.mak	Thu Feb 21 07:50:31 2019
+++ CPP/Build.mak	Tue Apr 28 21:24:54 2020
@@ -16,7 +16,7 @@


 !IF "$(PLATFORM)" == "x64"
-MY_ML = ml64 -Dx64 -WX
+MY_ML = ml64 -Dx64 -WX -nologo
 !ELSEIF "$(PLATFORM)" == "arm"
 MY_ML = armasm -WX
 !ELSE
@@ -137,6 +137,8 @@

 all: $(PROGPATH)

+$(OBJS_NEEDING_PCH): $O\StdAfx.obj
+
 clean:
 	-del /Q $(PROGPATH) $O\*.exe $O\*.dll $O\*.obj $O\*.lib $O\*.exp $O\*.res $O\*.pch $O\*.asm

@@ -146,7 +148,7 @@
 	if not exist "$O/asm" mkdir "$O/asm"

 $(PROGPATH): $O $O/asm $(OBJS) $(DEF_FILE)
-	link $(LFLAGS) -out:$(PROGPATH) $(OBJS) $(LIBS)
+	link $(LFLAGS) -out:$(PROGPATH) $O\StdAfx.obj $(OBJS) $(LIBS)

 !IFNDEF NO_DEFAULT_RES
 $O\resource.res: $(*B).rc
