--- tools/build/src/engine/vswhere_usability_wrapper.cmd	Wed Dec  4 23:03:07 2019
+++ tools/build/src/engine/vswhere_usability_wrapper.cmd	Wed Apr 29 20:23:25 2020
@@ -30,7 +30,7 @@

 REM Visual Studio 2019 (16.X, toolset 14.2)
 set VSWHERE_LMT=-version "[16.0,17.0)"
-SET VSWHERE_ARGS=-latest -products * %VSWHERE_REQ% %VSWHERE_PRP% %VSWHERE_LMT%
+SET VSWHERE_ARGS=-latest -prerelease -products * %VSWHERE_REQ% %VSWHERE_PRP% %VSWHERE_LMT%
 for /f "usebackq tokens=*" %%i in (`vswhere %VSWHERE_ARGS%`) do (
     endlocal
 	echo Found with vswhere %%i
